.github/workflows/package-publish.yml

```diff
@@ -0,0 +1,81 @@
+name: Package build and publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'version'
+        required: true
+        type: string
+      npm_tag:
+        description: 'release tag'
+        required: false
+        type: string
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          submodules: 'recursive'
+          fetch-depth: 0
+      - name: Enable corepack
+        run: corepack enable
```

Comment:
yarn binary is already included in the repo (https://github.com/sendbird/sendbird-uikit-react/tree/main/.yarn/releases)
```suggestion
```

Reply:
https://github.com/sendbird/sendbird-uikit-react/pull/1099/commits/e71179cba1e2d2851ff95a25b40ff72740d6da02#diff-771047455c9ee791192f052e9028c3c0b1acc9f00e6d8cc38dfe2b15e6c13fe8L24